python/tests/test_docker.py

```diff
@@ -84,6 +88,9 @@ def test_docker():
             logfile.write('No $AMBASSADOR_DOCKER_IMAGE??\n')
         else:
             if docker_start(logfile):
+                logfile.write("Demo started, sleeping 10 seconds just in case...")
+                time.sleep(10)
```

Comment:
Maybe we could have 2 checks? Fail fast first, then wait and see if the container remains healthy?